android/src/main/java/com/reactnativebledidcomm/central/CentralManager.kt

```diff
@@ -45,6 +45,20 @@ class CentralManager(private val context: ReactContext) {
         this.indicationCharacteristicUUID = indicationCharacteristicUUID
     }
 
+    fun shutdownCentral() {
+        try {
+            this.stopScan()
+        } catch (e: CentralManagerException.NotScanning) {
+            // Not Scanning
+        } finally {
+            this.serviceUUID = null
+            this.writeCharacteristicUUID = null
+            this.indicationCharacteristicUUID = null
+            this.connectedPeripheral = null
+            this.discoveredPeripherals.clear()
```

Comment:
Does clear just remove the connections from memory, or actually break the channel?

Reply:
it clears the array i think (from memory).

Reply:
it clears the array i think (from memory).

Reply:
actually for either do you have a preferred way of what is supposed to happen here?

Reply:
@blu3beri